src/app/games/coerceo/coerceo.component.ts

```diff
@@ -47,11 +47,11 @@ export class CoerceoComponent extends TriangularGameComponent<CoerceoRules,
         this.rules = CoerceoRules.get();
         this.node = this.rules.getInitialNode();
         this.availableAIs = [
-            new Minimax('Piece > Threat > Tiles Minimax',
+            new Minimax('Pieces > Threats > Tiles',
                         this.rules,
-                        new CoerceoPiecesThreatTilesHeuristic(),
+                        new CoerceoPiecesThreatsTilesHeuristic(),
                         new CoerceoOrderedMoveGenerator()),
-            new Minimax('Minimax', CoerceoRules.get(), new CoerceoHeuristic(), new CoerceoMoveGenerator()),
+            new Minimax('Minimax', this.rules, new CoerceoHeuristic(), new CoerceoMoveGenerator()),
```

Comment:
Euh du coup p-e pas en nommer un minimax si y a plusieurs minimaxes non ?
Il est dummy lui ?

Reply:
Captures and Freedom !

Reply:
pour le futur: interdire les minimax sans nom spécifique, car c'est toujours possible de les nommer